cogs/scammer.py

```diff
@@ -154,14 +155,27 @@ def check(m):
     @scammer.command(name="check", description="check a Minecraft username against our scammer list", usage="[username]")
     async def check(self, ctx, username:str):
         name, uuid = await fetch_uuid_uname(username)
+        cs = aiohttp.ClientSession()
+        jerry_resp = await cs.get("https://raw.githubusercontent.com/skyblockz/pricecheckbot/master/scammer.json")
+        jerry_data = await json.loads(await jerry_resp.text())
```

Comment:
Please use this style for http requests: https://github.com/Skybot-dev/Skybot-rewrite/blob/9d3f106423f132772b6b00602a799ad486b5c03a/utils/skypy/skypy.py#L370-L371